client_edit_modal.php

```diff
@@ -81,10 +100,26 @@
                                     <div class="input-group-prepend">
                                         <span class="input-group-text"><i class="fa fa-fw fa-globe"></i></span>
                                     </div>
-                                    <input type="text" class="form-control" name="website" placeholder="ex. google.com" value="<?php echo $client_website; ?>">
+                                    <input type="text" class="form-control" name="website" placeholder="ex. google.com"
+                                        value="<?php echo $client_website; ?>">
+                                </div>
+                            </div>
+
+                            <div class="form-group">
```

Comment:
I wanted to use the following switch from AdminLTE, but could never figure out how to get it to show as checked.
if someone else wants to take a crack at it...

```
                                <div class="input-group">
                                    <div class="custom-control custom-switch">
                                        <input class="custom-control-input" id="customSwitch1" type="checkbox" name="lead" value="1"<?php if ($client_is_lead == 1) {
                                            echo " checked";
                                        } ?>><label class="custom-control-label" for="customSwitch1">Is Lead?</label>
                                    </div>
                                </div>
```